### PR TITLE
Add: Allow Unlicense/MIT license

### PR DESCRIPTION
**Pull request by jjnicola**
## What
Add: Allow Unlicense/MIT license

## Why
Required by `walkdir` 

```
Licenses
  Warning: 
  The validity of the licenses of the dependencies below could not be determined. Ensure that they are valid SPDX licenses:
  rust/Cargo.lock » walkdir@2.4.0 – License: Unlicense/MIT
  Error: Dependency review could not detect the validity of all licenses.
```

## Checklist

- [ ] Tests

### CODE REVIEW

File: dependency-review/action.yml [hunk 1]
```diff
@@ -42,4 +42,5 @@ runs:
           MIT OR Apache-2.0,
           MIT AND Python-2.0,
           (Apache-2.0 AND BSD-3-Clause) OR (Apache-2.0 AND MIT),
-          (MIT OR Apache-2.0) AND Unicode-DFS-2016
+          (MIT OR Apache-2.0) AND Unicode-DFS-2016,
+          Unlicense/MIT
```
Comment (y0urself) [inline dependency-review/action.yml:46]: @jjnicola I don't think this is in the correct format and can not be recognized by the action ...

Comment (y0urself) [reply dependency-review/action.yml:46]: Correct! Nevertheless, this can not be "fixed" on our side!

**Comment by y0urself** [comment]
@jjnicola did you get approval for this license?

**Comment by y0urself** [comment]
@greenbone/scanner-maintainers is this still required? Did you get approval?

**Comment by nichtsfrei** [comment]
what approval? we still need that notation https://github.com/BurntSushi/walkdir/blob/master/Cargo.toml#L12 , yes.

It is fair to assume that the author doesn't seem to care about license notations as there is a PR open since 2021:
https://github.com/BurntSushi/walkdir/pull/155

**Pull request closed without merging**